src/screens/WasteCollectionScreen.js

```diff
@@ -441,7 +445,11 @@ export const WasteCollectionScreen = ({ navigation, route }) => {
           ]}
         >
           <Wrapper noPaddingBottom>
-            <Button title={wasteTexts.exportButton} notFullWidth onPress={triggerExport} />
+            <Button
+              title={wasteTexts.exportButton}
+              notFullWidth
+              onPress={() => triggerExport({ selectedTypes })}
```

Comment:
i guess that we can use `waste.selectedTypeKeys` here to avoid `Object.keys(selectedTypes).filter((key) => selectedTypes[key])` in `useTriggerExport` hook.

Reply:
Yes, that's right. I didn't see that. I think it's better now.